app/models/work.rb

```diff
@@ -51,4 +51,15 @@ def purl
 
     File.join(Settings.purl_url, T.must(druid).delete_prefix('druid:'))
   end
+
+  sig { params(edtf: T.nilable(T.any(EDTF::Interval, Date))).void }
```

Comment:
Was this ever working properly? Or did it fall through the cracks when we switched over to form-based validation?

Reply:
It's because the EDTF gem uses monkeypatching on the Date object rather than creating it's own object types. 😢

Reply:
Oh boy.